src/serializer.js

```diff
@@ -5,6 +5,9 @@
 **
 ** Bug: can't serialize  http://data.semanticweb.org/person/abraham-bernstein/rdf
 ** in XML (from mhausenblas)
+**
+** Todo: repeatable order of statments, with rdf:type as first pred.
```

Comment:
Is this TODO (and the above "Bug:") still relevant? It could get forgotten if we don't file an issue